pkg/util/deployment/deployment.go

```diff
@@ -130,8 +130,8 @@ func GetPodTemplateSpecHash(template api.PodTemplateSpec) uint32 {
 // Returns the sum of Replicas of the given replication controllers.
 func GetReplicaCountForRCs(replicationControllers []*api.ReplicationController) int {
 	totalReplicaCount := 0
-	for _, rc := range replicationControllers {
-		totalReplicaCount += rc.Spec.Replicas
+	for i := range replicationControllers {
+		totalReplicaCount += replicationControllers[i].Spec.Replicas
```

Comment:
Was thinking that since go reuses memory address in for loop, we should use index instead. However after looking closer, I find we don't have this problem in this loop since it's ranging []*api.ReplicationController but not taking address of the value like what we see in the other two PRs. Close this.